_data/investing.yml

```diff
@@ -33,6 +33,12 @@ websites:
       exceptions:
           text: "2FA only available on investment accounts. Not available for 401K plans."
 
+    - name: HealthEquity
+      url: https://www.healthequity.com/
```

Comment:
Not sure, it works for me:

```
$ curl -I https://www.healthequity.com/
HTTP/1.1 302 Found
Cache-Control: private
Content-Length: 139
Content-Type: text/html; charset=utf-8
Location: /HE.aspx?ReturnUrl=%2f
Server: Microsoft-IIS/7.5
X-Powered-By: ASP.NET
Access-Control-Allow-Credentials: true
X-UA-Compatible: IE=edge
Date: Tue, 04 Aug 2015 02:54:14 GMT
Set-Cookie: ARRAffinity2=!su85XZCtyqaRx/486aLBj3Hq8XsL+GJgByW2bdJJvauMBF485Yiiah1gR8MiUlMjjRFecOfzgw8B20=;HttpOnly;Secure;Path=/;Version=1;
```

Slim possibility they restrict requests to US IP addresses?  That'd be screwed up, but then again they don't support tfa either, so.... :) Can you get to their investor site, http://ir.healthequity.com/ ?